server/backend/mlx.py

```
@@ -191,7 +202,292 @@ def format_chat_messages_for_runner(
+        model=model,
+        status=status,
+        object="response",
+        error=error,
+        output=output,
+        usage=usage,
+    )
+    if metrics:
+        try:
+            resp.metadata["metrics"] = metrics
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)